engine/table/src/test/java/io/deephaven/engine/table/impl/PartitionedTableTest.java

```diff
@@ -851,6 +851,7 @@ public void testExecutionContext() {
                 newExecutionContextNugget(table, src -> src.select("K = queryScopeVar", "indices", "intCol")),
                 newExecutionContextNugget(table, src -> src.view("K = queryScopeVar", "indices", "intCol")),
                 newExecutionContextNugget(table, src -> src.where("intCol > queryScopeFilter")),
+                newExecutionContextNugget(table, src -> src.update("X = 0", "Y = X")),
```

Comment:
- Do you think we need/want a unit test for `DeferredViewTable`?
- Did you try the reported test case from Chip, as an extra confirmation step?

Reply:
Chip's test case still doesn't work because `drop_columns` and `merge` are not on TableOperations. However, it runs with modifications as follows:
```python
from deephaven import time_table

t1 = time_table("00:00:01").update(["Label = ii%2", "X = ii%2 == 0 ? cos(0.1*ii) : sin(0.01*ii)"])

t2 = t1.partition_by(["Label"]).proxy() \
    .update(["Data=X_.subVector(ii-14,ii)", "H=max(Data)", "L=min(Data)", "ATR=max(H-L, abs(H-X), abs(L-X))"]) \
    .target.merge()
```

I think DeferredViewTable tests, in general, would be great. I will add specifically a test for this. I'll create an issue for a follow up to test other deferred view table expectations.